src/common/commonutils/DaemonUtils.c

```diff
@@ -14,6 +40,11 @@ static int ExecuteSystemctlCommand(const char* command, const char* daemonName,
         OsConfigLogError(log, "ExecuteSystemctlCommand: invalid arguments");
         return EINVAL;
     }
+    else if (false == IsValidDaemonName(daemonName))
+    {
+        OsConfigLogError(log, "ExecuteSystemctlCommand: invalid daemon name '%s'", daemonName);

```

Comment:
same comment as above, might be useful to print that out here as well?

Reply:
Duplicate, closing.